superbench/executor/executor.py

```diff
@@ -15,19 +15,20 @@
 
 class SuperBenchExecutor():
     """SuperBench executor class."""
-    def __init__(self, sb_config, output_dir):
+    def __init__(self, sb_config, sb_output_dir):
         """Initilize.
 
         Args:
             sb_config (DictConfig): SuperBench config object.
-            output_dir (str): Dir for output.
+            sb_output_dir (str): SuperBench output directory.
         """
         self._sb_config = sb_config
-        self._output_dir = output_dir
+        self._sb_output_dir = sb_output_dir
```

Comment:
No need to keep `self._sb_output_dir` if already has `self._output_path`?

Reply:
the first one is the dir user inputs, e.g., `~/log`, it may refer to different paths on control node and managed nodes, e.g., different user name.
the second one refers to absolute path on control node, but the dir should be resolved again on managed nodes.